### PR TITLE
Making proxy work

### PR DESCRIPTION
**Pull request by nicola**
- [x] Proxy working
- [x] basic test
- [x] further tests
- [x] make sure cors works on proxy requests

### CODE REVIEW

**Comment by nicola** [comment]
Interesting travis might have some proxy for external resources, this needs further testing

File: lib/ldp.js [hunk 1]
```diff
@@ -53,8 +53,8 @@ function LDP(argv) {
   }
 
   ldp.pathFilter = regexp().start(ldp.mount).toRegExp();
-  ldp.xssProxy = argv.xssProxy;
-  ldp.proxyFilter = regexp().start(ldp.xssProxy).toRegExp();
+  ldp.proxy = argv.proxy;
+  ldp.proxyFilter = regexp().start(ldp.proxy).toRegExp();
```
Comment (deiu) [inline lib/ldp.js]: Quick question: what does proxyFilter do, and why do we need a filter for proxy?

**Comment by deiu** [comment]
r+

File: lib/ldp.js (hunk 1 above)
Comment (nicola) [inline lib/ldp.js]: Good question, removing that variable. It created a regex based on `ldp.proxy`, however express handles paths as regexp already, so no need for that

**Pull request merged**